### PR TITLE
protobuf: update docs

### PR DESCRIPTION
**Pull request by ilgooz**
closes https://github.com/mesg-foundation/core/issues/528

### CODE REVIEW

File: docs/api/core.md [hunk 1]
```diff
@@ -850,38 +850,21 @@ The request's data for `DeployService` API.
...
-      }
-    }
-  }
+  "url": "__SERVICE_GIT_URL__"
+}
+```
+or
+```json
+{
+  "url": "__SERVICE_GZIPPED_TAR_FILE_CHUNK__"
```
Comment (antho1404) [inline docs/api/core.md]: this should be chunk no ?

Comment (ilgooz) [reply docs/api/core.md]: yes, I'm still working in this PR. I'll request review after completed.

**Comment by ilgooz** [comment]
@antho1404 _ServiceLogs_ has missing docs also, would you like me to add it to this PR as well?

**Comment by antho1404** [comment]
yes can be good

File: protobuf/coreapi/api.proto [hunk 2]
```diff
@@ -281,7 +264,10 @@ message ListServicesRequest {
 // ```json
 // [{
 //   "service": {
+//     "id": "idX",
 //     "name": "serviceX",
+//     "description: "descriptionX",
```
Comment (antho1404) [inline protobuf/coreapi/api.proto]: `"` missing

Comment (antho1404) [reply protobuf/coreapi/api.proto]: same for status

File: protobuf/coreapi/api.proto [hunk 3]
```diff
@@ -328,7 +314,10 @@ message GetServiceRequest {
 // ```json
 // {
 //   "service": {
+//     "id": "idX",
 //     "name": "serviceX",
+//     "description: "descriptionX",
+//     "status: "statusX",
```
Comment (antho1404) [inline protobuf/coreapi/api.proto]: same here

File: protobuf/coreapi/api.proto [hunk 4]
```diff
@@ -358,29 +347,39 @@ message GetServiceReply {
 }
 
 // The request's data for `ServiceLogs` API.
+//
+// **Example**
+// ```json
+// {
+//   "serviceID": "__SERVICE_ID__",
+//   "dependencies": "__SERVICE_DEPENDENCIES__"
```
Comment (antho1404) [inline protobuf/coreapi/api.proto]: this is a `repeated string` so i'm not sure we can pass a string here, it should be an array

Comment (antho1404) [reply protobuf/coreapi/api.proto]: Don’t forget about that one or maybe it’s normal?

Comment (ilgooz) [reply protobuf/coreapi/api.proto]: this is also done, forgot add the comment about this :)

Comment (antho1404) [reply protobuf/coreapi/api.proto]: Are you sure you committed it ? I can't see it, I'm talking about replacing the example of `ServiceLogsRequest` with
```
{
  "serviceID": "__SERVICE_ID__",
  "dependencies": ["__DEPENDENCY_X__"]
}
```
Putting the array for the repeated string

Comment (ilgooz) [reply protobuf/coreapi/api.proto]: oh got it. do you think we should do same for the chunks(byte arrays)

**Comment by antho1404** [comment]
Just saw another issue on the documentation concerning the service `ListServicesReply` and `GetServiceReply`.
The service does not contains map anymore for events, outputs, tasks... it contains repeated data so this should also be updated.

Example ListServicesReply:

```json
{
  "services": [{
    "id": "idX",
    "name": "serviceX",
    "description": "descriptionX",
    "status": "statusX",
    "events": [{
      "key": "eventX",
      "data": [{
        "key": "dataX",
        "type": "String"
      }]
    }],
    "tasks": [{
      "key": "taskX",
      "inputs": [{
        "key": "foo",
        "type": "String"
      }],
      "outputs": [{
        "key": "outputX",
        "data": [{
          "key": "resX",
          "type": "String"
        }]
      }]
    }]
  }]
}
```

Example GetServiceReply:

```json
{
  "service": {
    "id": "idX",
    "name": "serviceX",
    "description": "descriptionX",
    "status": "statusX",
    "events": [{
      "key": "eventX",
      "data": [{
        "key": "dataX",
        "type": "String"
      }]
    }],
    "tasks": [{
      "key": "taskX",
      "inputs": [{
        "key": "foo",
        "type": "String"
      }],
      "outputs": [{
        "key": "outputX",
        "data": [{
          "key": "resX",
          "type": "String"
        }]
      }]
    }]
  }
}
```


To verify with the API but it should be something like that

File: protobuf/coreapi/api.proto (hunk 4 above)
Comment (ilgooz) [reply protobuf/coreapi/api.proto]: done

Comment (ilgooz) [reply protobuf/coreapi/api.proto]: actually, the data we're sending is a [byte slice](https://github.com/mesg-foundation/core/blob/dev/protobuf/coreapi/api.pb.go#L726). _bytes_ in protobufs is equal to a byte array and a _repeated bytes_  would be the equivalent of _array of byte arrays_. I think it makes sense to update them as arrays

**Comment by antho1404** [comment]
Updated the list service reply that needed a services attribute that is a list of services
Also updated the cli documentation with some missing documentation

**Pull request merged**